sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/changestreams/action/DataChangeRecordAction.java

```diff
@@ -85,6 +85,7 @@ public Optional<ProcessContinuation> run(
 
     final String token = partition.getPartitionToken();
     LOG.debug("[{}] Processing data record {}", token, record.getCommitTimestamp());
```

Comment:
there is already debug log here. One can set `--sdkHarnessLogLevelOverrides='{\"org.apache.beam.sdk.io.gcp.spanner.changestreams.action.DataChangeRecordAction\":\"DEBUG\"}'"` to enable it. See https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/options/SdkHarnessOptions.SdkHarnessLogLevelOverrides.html